ush/get_extrn_mdl_file_dir_info.sh

```diff
@@ -814,6 +824,10 @@ has not been specified for this external model:
 
       arcv_fns="${arcv_fns}gfs_pgrb2"
 
+    elif [ "${fv3gfs_file_fmt}" = "netcdf" ]; then
+
+      arcv_fns="${arcv_fns}gfs_pgrb2"
+
```

Comment:
Should this not be a netcdf file? If not, please leave a comment as to why since this is confusing logic, and consider putting it into the previous `elif` statement so there isn't duplicate information here.